src/core/TreeProcessor.js

```diff
@@ -68,6 +69,8 @@ getJasmineRequireObj().TreeProcessor = function() {
       } else {
         var hasExecutableChild = false;
 
+        willProcessNode(node);
```

Comment:
`willProcessNode` doesn't really mean anything to me. I think it should be something like `orderChildren`, possibly just taking the children in. I would prefer to be more specific to start with, especially with an internal interface, until we actually find something else that would need to plug in here.